modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -283,6 +287,143 @@ For production {product-title} clusters on which you want to perform installatio
 ====
 endif::private[]
 
+ifdef::restricted[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+    ibm-cloud: {}
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16 <6>
+  networkType: OVNKubernetes <7>
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  ibmcloud:
+    region: us-east <1>
+    resourceGroupName: us-east-example-network-rg <8>
+    serviceEndpoints: <9>
+      - name: IAM
+        url: https://private.us-east.iam.cloud.ibm.com
+      - name: VPC
+        url: https://us-east.private.iaas.cloud.ibm.com/v1
+      - name: ResourceController
+        url: https://private.us-east.resource-controller.cloud.ibm.com
+      - name: ResourceManager
+        url: https://private.us-east.resource-controller.cloud.ibm.com
+      - name: DNSServices
```

Comment:
we only support DNSSerivce endpoint override, but not CIS, so the IBMCloud install in a restrict network has to be a private cluster together, right? But the doc, it is saying it is optional.
```
Optional: Set the publishing strategy to `Internal`:
```

Reply:
> we only support DNSSerivce endpoint override, but not CIS, so the IBMCloud install in a restrict network has to be a private cluster together, right? But the doc, it is saying it is optional.
> 
> ```
> Optional: Set the publishing strategy to `Internal`:
> ```

You could use a publishing strategy of `external` for a restricted/disconnected network install. In which case, a private service endpoint override for CIS would be needed. This use case seems odd, but theoretically possible.

Reply:
>  I need confirm my statement and verify availability of a private CIS service endpoint.

On https://cloud.ibm.com/vpc-ext/provision/endpointGateway, I only can see a private DNSService endpoint, but no CIS.